README.md

```diff
@@ -1,35 +1,54 @@
-# Commands to run the server
-
-* 1) Run database locally
-    mongod or sudo mongod
-    * In case it's not installed, install mongo: https://docs.mongodb.com/v3.2/administration/install-community/
-
-
-* 2) Install server dependencies:
+# Commands to install and run the backend application
+
+* Download and install node version 12.18.0 https://nodejs.org
+  
+* Install “nodemon” by running this command : npm install -g nodemon
+  
+* Download and install MongoDB community Edition , note the connection properties  -     https://docs.mongodb.com/manual/administration/install-community/
+  
+* Download and install MongoDBCompass for the UI based DB Management - https://www.mongodb.com/products/compass
```

Comment:
Here also let's not be specific. Let's say 'You can install UI app like mongoDB Compass for easiness of the DB  management' or something like that.

Reply:
WIll be updated